lib/types/subgraph.ts

```diff
@@ -1,33 +1,42 @@
 export interface KlimaRetire {
```

Comment:
Can this type and `DigitalCarbonCredit` be generated from the API?

Reply:
I will add this to the GH issue. It should be straightforward once these queries are moved to the api.